models/spring-ai-ollama/src/test/java/org/springframework/ai/ollama/OllamaChatModelIT.java

```diff
@@ -58,18 +59,18 @@
 @Disabled("For manual smoke testing only.")
 class OllamaChatModelIT {
 
-	private static String MODEL = "mistral";
+	private static final String MODEL = OllamaModel.MISTRAL.getName();
 
 	private static final Log logger = LogFactory.getLog(OllamaChatModelIT.class);
 
 	@Container
-	static OllamaContainer ollamaContainer = new OllamaContainer("ollama/ollama:0.1.32");
+	static OllamaContainer ollamaContainer = new OllamaContainer("ollama/ollama:0.2.8");
```

Comment:
The Ollama integration test setup could use some cleanup and refactoring, but let's keep that for another PR to keep the scope focused here.